landoapi/hgexports.py

```diff
@@ -620,37 +620,42 @@ def run_diff_checks(self) -> list[str]:
 
 @dataclass
 class PushCheck:
```

Comment:
I guess you didn't end up renaming this? Your comment seemed to imply that you were going to do that.

Reply:
My mistake, it's renamed now.